core/src/consensus/stake/action_data.rs

```diff
@@ -456,6 +456,59 @@ impl IntermediateRewards {
     }
 }
 
+#[derive(Default, Debug, PartialEq)]
+pub struct IntermediateRewardsV1 {
```

Comment:
(just a comment) I understand that the operational semantics are changed without changing the format. However, it triggers me somehow.